cmd/api/src/api/registration/v2.go

```diff
@@ -285,7 +285,8 @@ func NewV2API(cfg config.Configuration, resources v2.Resources, routerInst *rout
 
 		// Datapipe API
 		routerInst.GET("/api/v2/datapipe/status", resources.GetDatapipeStatus).RequireAuth(),
-		//TODO: Update the permission on this once we get something more concrete
+		//TODO: Update the permission on these once we get something more related to sql db
```

Comment:
The state is related to SQL, but analysis itself I believe is about data in the graph DB, so the GraphDB permissions seem more appropriate than a SQL one. I suspect the original comment was referring to a permission specifically related to analysis.